lib/bolt/project.rb

```diff
@@ -227,7 +227,10 @@ def validate
           raise Bolt::ValidationError, "The project '#{name}' will not be loaded. The project name conflicts "\
             "with a built-in Bolt module of the same name."
         end
-      else
+      elsif name.nil? &&
+            (File.directory?(plans_path) ||
+            File.directory?(@path + 'tasks') ||
+            File.directory?(@path + 'files'))
```

Comment:
Why are we including the `files/` directory in this check? I know `tasks/` and `plans/` make sense, since you can reference content in those directories from the CLI, but I don't think you can for `files/` (unless I'm missing a command).

Reply:
You could reference a file from the project in a manifest being applied, for (very contrived) example:
```
bolt apply -e " file { '/tmp/foobar': content => file('myproject/service.conf'), ensure => present }"
```
Or more likely from a manifest file you specify with `bolt apply`.

I think we opted not to include manifests for the reason you said, that usually if you're consuming manifests it's probably in a plan (in which case you'd get the error). We could add `manifests/` too to err on the side of zealously warning, in case people are consuming the project from a module on the modulepath or something.

Reply:
I'm not really worried about it, and it's easy enough to add to the check later if needed. :) I was more just curious why `files/` was included, and that example makes perfect sense!